client/package.json

```
@@ -40,7 +40,7 @@
     "@solana/spl-token-metadata": "^0.1.2",
     "@solana/wallet-adapter-base": "^0.9.22",
     "@solana/wallet-adapter-react": "^0.15.32",
-    "@solana/web3.js": "*",
+    "@solana/web3.js": "^1.92.1",
```

Can we keep the version as is here and only make changes to the lock file?
```suggestion
    "@solana/web3.js": "*",
```